contracts/EventStore.sol

```diff
@@ -84,7 +62,6 @@ contract EventStore is Killable {
       StringValue: _string
     });
     SOLIDITY_EVENT_PROPERTY(_eventIndex, _eventPropertyIndex, _name, _type, _address, _uint, _string);
-    solidityEventCount += 1;
     return solidityEventCount;
```

Comment:
@OR13 This should either return a bool or the event property count. I'm leaning towards bool because we know the event property count.